Lib/xml/etree/ElementTree.py

```diff
@@ -194,6 +194,13 @@ def copy(self):
         original tree.
 
         """
+        warnings.warn(
+            "elem.copy() is deprecated. Use copy.copy(elem) instead.",
+            DeprecationWarning
```

Comment:
Not sure if we should issue a `DeprecationWarning` straight away (or a `PendingDeprecationWarning` first), although I personally wouldn't mind, given that the C implementation has shadowed this method for years (in CPython, that is).
@serhiy-storchaka, what do you think?

Reply:
FWIW, `copy()` does not appear in the documentation, either.

Reply:
Also, I can't find any record of `copy` ever having been available in the C implementation; it looks like it was `__copy__` from the beginning.

The first appearance of `copy` in the Python implementation was in 351c67416ba4451eb3928fa0b2e933c2f25df1a3 (first released in Python 3.1 and 2.7.4), which deemed it "experimental".